sage-spkg-info.py

```diff
@@ -0,0 +1,156 @@
+import argparse
+import subprocess
+import os
+
+# To Parse input arguments
+def parse_args():
+    parser = argparse.ArgumentParser(description="Fetch and format Sage package information.")
+    parser.add_argument('pkg_base', help="Package base name")
+    parser.add_argument('--output_dir', help="Directory for output files", default=None)
+    parser.add_argument('--output_rst', action='store_true', help="Output in reStructuredText format")
```

Comment:
Such command-line options are best kept in "kebab-case", i.e., using dashes, not underscores: `--output-dir`, `--output-rst`